LocationServices/LocationServices/Extensions/LanguageManager.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import UIKit
+
+var bundleKey: UInt8 = 0
+
+class LanguageManager {
+    static let shared = LanguageManager()
+    private(set) var translations: [String: [String: String]] = [:]
+
+    var currentLanguage: String {
+        get {
+            return Locale.currentAppLanguageIdentifier()
+        }
+        set {
+            UserDefaultsHelper.save(value: [newValue], key: .AppleLanguages)
+            loadStrings()
+            GeneralHelper.reloadUI()
+        }
+    }
+    
+    func loadStrings(from file: String = "Localizations") {
```

Comment:
What does `Localizable.xcstrings` do? I can't load the diff in Github so I'm curious. Why is this separate from `StringConstant`?